ground_control_station/StatusNew[] StatusArray